### PR TITLE
Added .taskcluster.yml validation tool

### PR DESCRIPTION
**Pull request by lotas**
Experimental for now.
Intention is to allow devs to test their `.taskcluster.yml` before committing it, or detect possible issues with event configuration.

This might later be merged together with "Github Quickstart" but for now is  a standalone view

image

image

### CODE REVIEW

File: ui/src/App/routes.jsx [hunk 1]
```diff
@@ -130,6 +133,10 @@ export default [
     component: Quickstart,
     path: '/quickstart',
   },
+  {
+    component: TcYamlDebug,
```
Comment (lotas) [inline ui/src/App/routes.jsx:137]: not linked in the menu for now..

File: ui/src/views/TcYamlDebug/test-payload.js
Comment (lotas) [inline ui/src/views/TcYamlDebug/test-payload.js:1]: just a dump of few webhook events, that would be used to render tc.yml

File: ui/package.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
     "hterm-umdjs": "^1.4.1",
     "is-absolute-url": "^4.0.1",
     "js-yaml": "^4.0.0",
-    "json-e": "^4.0.0",
+    "json-e": "^4.5.2",
```
Comment (lotas) [inline ui/package.json:60]: to have latest '$switch - $default' syntax

**Review by matt-boris — APPROVED** [review]
Awesome!!

**Pull request merged**